pkgs/by-name/cr/craft-docs/package.nix

```diff
@@ -0,0 +1,44 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchurl,
+  undmg,
+}:
+
+stdenvNoCC.mkDerivation (finalAttrs: {
```

Comment:
You can omit the finalAttrs wrapper, since it is not used.

Reply:
Good to know. I made this by copying and pasting from another package before I understood nix very well, so I'm sure I got a bunch wrong.